mediathread/main/tests/test_views.py

```diff
@@ -454,7 +454,7 @@ def test_migrate_materials_sample_course(self):
 
         the_json = json.loads(response.content)
         self.assertEquals(the_json['course']['title'], 'Sample Course')
-        self.assertEquals(len(the_json['assets']), 1)
+        self.assertEquals(len(the_json['assets']), 2)
```

Comment:
This test changed when I made the change from populating the asset context from `notes` to `assets`. I'm looking further into this to find out exactly why we're now getting two assets here instead of one..

Reply:
Okay, see commit 89f1d7794266be8ff1c31701a56f7d9188fd7b56. I've changed the test here back to its original state.

In the MigrateMaterialsView, notes was excluding the flv_pseudo type, but not assets, which is now used as well. So, I've added an exclude case here. It seems right to me, but I'm not really familiar with all these types, why flv_pseudo is getting excluded here in the first place.